src/poliastro/core/elements.py

```diff
@@ -52,12 +112,27 @@ def coe2rv(k, p, ecc, inc, raan, argp, nu):
 
 @jit
 def coe2mee(p, ecc, inc, raan, argp, nu):
-    """Converts from classical orbital elements to modified equinoctial
+    r"""Converts from classical orbital elements to modified equinoctial
     orbital elements.
 
     The definition of the modified equinoctial orbital elements is taken from
     [Walker, 1985].
 
+    The modified equinoctial orbital elements are a set of orbital elements that are useful for
+    trajectory analysis and optimization. They are valid for circular, elliptic, and hyperbolic
+    orbits. These direct modified equinoctial equations exhibit no singularity for zero
+    eccentricity and orbital inclinations equal to 0 and 90 degrees. However, two of the
+    components are singular for an orbital inclination of 180 degrees.
+
+    .. math::
+
+        p = a(1-e^2) \\
```

Comment:
Do you think we can try to use an `align` environment instead, to align all the `=` equal signs? https://math.meta.stackexchange.com/a/5024/24849